docs/manage/troubleshoot/troubleshoot.md

```diff
@@ -89,6 +92,9 @@ If not follow the steps for the standalone version.
 {{% /tab %}}
 {{% tab name="macOS" %}}
 
+`viam-server` writes logs to STDOUT and does not store them in a file on your machine.
```

Comment:
The Linux instruction makes sense but for the mac one, maybe clarify where STDOUT goes by default so there's more of a distinction between default behavior versus what -debug does?